website/docs/r/compute_http_health_check.html.markdown

```diff
@@ -1,21 +1,31 @@
+<!---
+ ----------------------------------------------------------------------------
+
+     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+
+ ----------------------------------------------------------------------------
+
+     This file is automatically generated and manual changes will be
+     clobbered when the file is regenerated.
+
+     Please read more about how to change this file in
+     .github/CONTRIBUTING.md.
+
+ ----------------------------------------------------------------------------
+--->
 ---
```

Comment:
I don't know if this is going to cause a problem for the hashicorp parser, but we actually need a newline before this line if this is strict markdown.  What do you think?

Reply:
Done